tests/cupyx_tests/scipy_tests/ndimage_tests/test_filters.py

```diff
@@ -154,7 +163,7 @@ def _get_weights(self, xp):
                            'minimum_filter1d', 'maximum_filter1d'):
             return self.ksize
 
-        if self.filter == 'uniform_filter':
+        if self.filter in ('uniform_filter'):
```

Comment:
Q: Missing a comma?

Reply:
Thanks. It looks like this whole `if` statement was no longer needed due to some other refactoring. I removed it in the latest commit